examples/charts/README.md

```diff
@@ -21,6 +21,12 @@ There are two subdirectories:
 
 * `file` examples that output to static HTML files
 
+* `server`
+
+The example code for charts using Jupyter/Ipython notebooks is located under "howto"
+../howto/charts
```

Comment:
I think if you use the markdown syntax, i.e. `[text](address)` this will render as a nice link